packages/machine/test/unit/cf-operations/set-state.spec.ts

```diff
@@ -4,6 +4,7 @@ import { ethers } from "ethers";
 import { OpSetState } from "../../../src/middleware/protocol-operation";
 
 import {
+  constructContractCall, TEST_APP_INSTANCE,
```

Comment:
This should get caught by the linter - need to be consistent in terms of importing, so all imports should be on their own line.

Reply:
Yeah weird, I ran a lint:fix on it too